extension-manifest-v3/src/background/helpers.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Ghostery Browser Extension
+ * https://www.ghostery.com/
+ *
+ * Copyright 2017-present Ghostery GmbH. All rights reserved.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0
+ */
+
+chrome.runtime.onMessage.addListener((msg, sender, sendResponse) => {
+  switch (msg.action) {
+    case 'getCurrentTab':
```

Comment:
Firstly, I also moved here a message receiver for opening a new tab, so there were more of them. 

I think a switch is clearer here (as we can introduce more of the helper messages). For now, it could be if/else, but the switch does not introduce any perf issue, so I would keep it.